shared/macros-oval.jinja

```diff
@@ -1,4 +1,16 @@
-{{%- macro oval_check_config_file(msg='', path='', prefix_regex='^\s*', parameter='', separator_regex='(?-i)[\s]+', value='', default_pass='', application='', match_whole_word='') -%}}
+{{#
+  High level macro which checks for a particular set of parameter plus value in a configuration file.
+  This function can take eight values:
+    - path (String): Path to the configuration file to be checked.
+    - prefix_regex (String): Regular expression to be used in the beginning of the OVAL text file content check.
+    - parameter (String): The parameter to be checked in the configuration file.
+    - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
+    - separator_regex (String): Regular expression to be used as the separator of parameter and value in a configuration file.
```

Comment:
Should it mention that if white spaces are allowed, the regex should account for it?

Reply:
I think so. I'll fix it.